packages/api-catalog-spa/src/router/links.ts

```diff
@@ -0,0 +1,16 @@
+export enum ApiCatalogLinks {
+  HomePage = '/',
+  ListPage = '/apis',
+  AddNewApiPage = '/apis/new',
+  ApiDescriptionPage = '/apis/:id',
+  ApiEditPage = '/apis/:id/edit',
```

Comment:
```suggestion
  ApiEditPage = '/apis/:id/edit',
```

WIth the slug the api should do the operations. mongodb id no need to to show it

Reply:
@riginoommen That id and mongodb id has no relation it for frontend id positioning. But yah Mongo id is used as we discussed. 

I have added nanoid(6) as slug  for now